src/Uno.UI.Tests/Windows_UI_XAML_Controls/UserControlTests/UserControl_TopLevelBinding_UserControl.xaml.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace Uno.UI.Tests.Windows_UI_XAML_Controls.UserControlTests
+{
+	/// <summary>
+	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// </summary>
+	public sealed partial class UserControl_TopLevelBinding_UserControl : UserControl
+	{
+		public UserControl_TopLevelBinding_UserControl()
+		{
+			this.InitializeComponent();
+		}
+	}
+
+	public static class UserControl_TopLevelBinding_AttachedProperty
+	{
+		public static int GetMyProperty(DependencyObject obj)
+		{
+			return (int)obj.GetValue(MyPropertyProperty);
+		}
+
+		public static void SetMyProperty(DependencyObject obj, int value)
+		{
+			obj.SetValue(MyPropertyProperty, value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'obj' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5619051)